study-builder/studies/mbc/study.conf

```diff
@@ -37,7 +37,18 @@
     "guid": "CMIPEPPERADMINUSER"
   },
 
-  "studyDetails": [],
+  "studyDetails": [
+    {
+      "language": "en",
+      "name": "Metastatic Breast Cancer Project",
+      "summary": null
+    },
+    {
+      "language": "es",
+      "name": "Metastatic Breast Cancer Project",
+      "summary": null
+    },
```

Comment:
Do we need the `es` translation if they're both identical?

Reply:
I think so, because the code looks it up by language and if it doesn't find it, I don't think it will automatically assume to grab the English one.

Reply:
Do you think this should be tracked in Jira? From my understanding, this differs from our usual translation behavior of falling back to the default language.

Reply:
Yeah, I agree! I think if we didn't find the "preferred language" one, we should fallback to the study's "default language" and try to look for "study details" for that language. And if that fails also, then we fallback to "study dto name". I made a comment about this in @ssettipalli's PR: https://github.com/broadinstitute/ddp-study-server/pull/872#discussion_r669087343. Sampath, did we create a ticket to track this?

Reply:
In any case, I think there shouldn't be any issues with the configurations we have in this PR. For Spanish, I recall that we also want to show the English study name, so we should be good-to-go.